docs/report-abuse.md

```diff
@@ -16,4 +21,4 @@ The following uses of our service are considered as malicious:
 - Criminal activity
 
 
-Following your report, we will investigate and take the necessary actions if the abuse is indeed verified.
\ No newline at end of file
+In event when a case of abuse is verified. SimpleLogin reserve the right to terminate or suspend access to the Services immediately, without prior notice or liability, for any harmful behavior that includes but not limited to: sending spams, spreading viruses, repeatedly marking emails from SimpleLogin as spam.
```

Comment:
I'd change it slightly to "If the abuse is verified,  SimpleLogin reserves the right to terminate or suspend access to our services immediately, without prior notice or liability, for any harmful behavior that includes but not limited to: sending spams, spreading viruses, repeatedly marking emails from SimpleLogin as spam."